src/components/CompareResults/CompareResultsTable.tsx

```diff
@@ -14,7 +14,6 @@ import CompareResultsTableRow from './CompareResultsTableRow';
 const tableHeaders: CompareResultsTableHeaders[] = [
   { id: 'platform', label: 'Platform', align: 'center' },
   { id: 'graph', label: 'Graph', align: 'center' },
-  { id: 'suite', label: 'Suite', align: 'left' },
```

Comment:
I didnt make any change here maybe it's the diff between staging and mocking ?

Reply:
yes, likely so. I'll create a patch to update the mocking branch to match the latest staging.